recipes/ckanweb-deploy-exts.rb

```diff
@@ -265,21 +265,29 @@
 				fi
 			EOS
 		end
+		if batchnode
+            # Run dataset require updates notifications at 7am and 7:15am on batch
+            file "/etc/cron.d/ckan-dataset-resource-visibility-notify-privacy-assessments" do
```

Comment:
This file should be removed if the appropriate plugin no longer exists. Compare lines 402 and 412.